src/collections/Form/Form.js

```diff
@@ -94,10 +94,14 @@ class Form extends Component {
   static Select = FormSelect
   static TextArea = FormTextArea
 
-  handleSubmit = e => {
-    const { action, onSubmit } = this.props
-    if (!action) e.preventDefault()
-    if (onSubmit) onSubmit(e, { ...this.props })
+  handleSubmit = (...args) => {
+    const { action } = this.props
+    const [e] = args
```

Comment:
@levithomason I've updated this part, too. Main idea to pass all args from third-party lib correctly.

Let me know what you think.